pkgs/tools/package-management/nixpkgs-review/default.nix

```diff
@@ -37,7 +37,8 @@ python3.pkgs.buildPythonApplication rec {
   meta = with lib; {
     description = "Review pull-requests on https://github.com/NixOS/nixpkgs";
     homepage = "https://github.com/Mic92/nixpkgs-review";
+    changelog = "https://github.com/Mic92/nixpkgs-review/releases/tag/${version}";
     license = licenses.mit;
-    maintainers = with maintainers; [ mic92 SuperSandro2000 ];
+    maintainers = with maintainers; [ figsoda mic92 SuperSandro2000 ];
```

Comment:
```suggestion
    maintainers = with maintainers; [ mic92 figsoda SuperSandro2000 ];
```

Reply:
any reasons to do this? I just sorted it alphabetically

Reply:
Maybe to indicate some sort of priority? But honestly I don't care here :)